Prettier

ajhyndman opened the pull request.

Jberlinsky commented on circle.yml:
```diff
@@ -0,0 +1,5 @@
+# Customize test commands
```
Can we remove this comment?

ajhyndman replied:
Sure.  Any particular reason?

Jberlinsky commented on src/components/App/App.js:
```diff
@@ -1,14 +1,11 @@
-    );
-  }
-}
+const App = ({ children }) => <div className="App">{children}</div>;
```
👍

jayce-leathers replied:
Agreed, definitely prefer the functional style

jayce-leathers commented:
Actually something just occurred to me. I think we should consider creating and maintaining documentation for our development process. That includes the purpose of and how we use tools like Prettier. We could just add that to the readme for now? Or start a wiki page? I think this will pay off in ease of on-boarding new devs later?

ajhyndman commented:
@jayce-leathers yeah, I think that's a great idea!  I think especially because we have a fairly distributed team (likely with some turnover 😛), there's a lot of incentive for good documentation.

I'll have a think, and if I don't come up with anything better I'll put something in the README.

ajhyndman commented:
@jayce-leathers I've started a Wiki, and added a Developer Tooling page with a brief motivation for each of the tools we've added so far.

https://github.com/civic-tech-lab/commonplace-frontend/wiki

How do you feel about this approach?  Let me know if you have any suggestions.

ajhyndman commented:
I'm going to go ahead and merge this first sequence of Pull Requests.

If anyone has further suggestions, feel free to open further issues, or otherwise comment elsewhere.
